keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c

```diff
@@ -0,0 +1,71 @@
+/* Copyright 2019 Garret G. (TheRoyalSweatshirt)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define BASE 0
+#define FN 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[BASE] = LAYOUT(
+		KC_7, KC_8, KC_9, \
+		KC_4, KC_5, KC_6, \
+		KC_1, KC_2, KC_3, \
+		MO(1), KC_0, KC_DOT \
+	),
+  
+	[FN] = LAYOUT(
+		KC_TRNS, KC_HOME, KC_PGUP, \
+		KC_TRNS, KC_END, KC_PGDN, \
+		KC_TRNS, KC_TRNS, KC_TRNS, \
+		KC_TRNS, KC_TRNS, KC_ENT \
+	)
+};
+
+#ifdef OLED_DRIVER_ENABLE
+void oled_task_user(void) {
```

Comment:
This may be a bit of an odd request, but could this also be added to the keyboard's `c` file, but set weakly?  (eg, with `__attribute__((weak))` right before it)?

This way, if compiling in the configurator, it still has the OLED code, but could be overridden by the keymap for those that want to customize it.